src/checkers/inference/dataflow/InferenceValue.java

```diff
@@ -221,4 +222,22 @@ private TypeMirror getGlbType(final CFValue other, final CFValue backup) {
 
         return underlyingType;
     }
+
+    @Override
+    public String toString() {
+        StringBuilder sb = new StringBuilder("InferenceValue{annotations=");
```

Comment:
The `annotations=` part is a bit misleading, as there is only one. Should this differ between a constant and a variable?